core/modules/css/KvInterface.h

```diff
@@ -42,18 +42,29 @@ namespace css {
 
 class KvInterface {
 public:
+
+    /**
+     * Sets a max length for kvKey.
+     * If it is changed it must also be changed in the
+     * CssData SQL Schema as well.
+     */
+    static const unsigned int MAX_KEY_LENGTH = 255;
+
     virtual ~KvInterface() {};
 
     /**
-     * Create a key/value pair.
-     * Throws KeyExistsError if the key already exists (or if any other problem,
-     * e.g., a connection error is detected).
+     * Create a slash-delimited key-value pair.
+     * Key must be shorter than MAX_KEY_LENGTH.
+     * If the parent key does not exist it will be created with an empty value.
+     * Throws KeyExistsError if the key already exists
```

Comment:
For Doxygen comments I'd prefer to have special doxygen markup like:

    @throws KeyExistsError if the key already exists